Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2
 
+WORKDIR /go/src/github.com/open-cluster-management/search-collector
+COPY . .
+RUN CGO_ENABLED=0 GOGC=25 go build -trimpath -o main main.go
```

Comment:
For reference, these are the resource requests set for each step of the build. We can modify this by changing our openshift release repo if necessary. The recommend/default are:
```
resources:
  '*':
    requests:
      cpu: 100m
      memory: 200Mi
```